assets/airports/kcvg.json

```diff
@@ -0,0 +1,4741 @@
+        "HYK"  : ["N37d57.98", "W84d28.35"],
+        "IIU"  : ["N38d06.21", "W85d34.65"],
+        "JABDU": ["N38d52.81", "W84d34.51"],
+        "JAKIE": ["N38d25.92", "W84d09.18"],
+        "JARSO": ["N39d35.50", "W83d54.70"],
+        "JAYKB": ["N38d38.83", "W82d25.77"],
+        "JBETH": ["N38d59.92", "W84d26.29"],
+        "JBNCH": ["N39d34.88", "W85d01.36"],
+        "JDUNN": ["N38d22.07", "W88d16.27"],
+        "JKUKA": ["N38d35.30", "W85d38.23"],
+        "JODUB": ["N38d45.75", "W82d53.46"],
+        "JOT"  : ["N41d32.79", "W88d19.10"],
+        "JRICO": ["N38d15.56", "W86d42.50"],
+        "KENLN": ["N38d27.45", "W84d30.47"],
+        "KNEDY": ["N38d39.66", "W84d00.81"],
+        "LOVEY": ["N39d12.81", "W85d24.52"],
+        "MACES": ["N40d00.22", "W86d43.38"],
+        "MEEKR": ["N39d24.80", "W85d08.14"],
+        "MIE"  : ["N40d14.24", "W85d23.64"],
+        "MIPIE": ["N38d56.01", "W84d48.81"],
+        "MISSN": ["N38d45.79", "W83d00.05"],
+        "MOIER": ["N39d06.65", "W84d05.49"],
+        "MUMIE": ["N39d04.31", "W85d01.88"],
+        "NADIR": ["N40d36.35", "W84d33.87"],
+        "NICID": ["N39d04.69", "W84d40.11"],
+        "NUZUM": ["N39d02.80", "W84d56.94"],
+        "OBK"  : ["N42d13.28", "W87d57.10"],
+        "ODABE": ["N37d39.66", "W83d41.86"],
+        "OGIRE": ["N39d24.88", "W84d32.50"],
+        "ORLEE": ["N39d42.67", "W83d38.04"],
+        "PACKO": ["N36d47.10", "W82d42.70"],
+        "PONIE": ["N38d19.83", "W85d03.28"],
+        "PXV"  : ["N37d55.70", "W87d45.74"],
+        "RHOMM": ["N38d45.88", "W83d31.35"],
+        "RID"  : ["N39d45.30", "W84d50.33"],
+        "RIKLE": ["N39d49.83", "W84d07.30"],
+        "ROCKT": ["N39d36.44", "W82d52.21"],
+        "ROD"  : ["N40d17.27", "W84d02.59"],
+        "SARGO": ["N38d40.13", "W85d27.80"],
+        "SHB"  : ["N39d37.95", "W85d49.46"],
+        "SHEAN": ["N40d24.22", "W85d01.34"],
+        "SILKS": ["N38d28.00", "W84d45.10"],
+        "SKINN": ["N37d49.50", "W86d04.62"],
+        "SLUBY": ["N39d02.63", "W84d11.60"],
+        "STEVY": ["N38d49.19", "W80d07.16"],
+        "TAPIE": ["N38d54.75", "W84d30.05"],
+        "TAYOS": ["N37d38.37", "W83d35.29"],
+        "TIGRR": ["N39d29.40", "W84d03.18"],
+        "TORIO": ["N40d14.95", "W86d52.56"],
+        "TRFWA": ["N38d01.03", "W84d47.43"],
+        "TTH"  : ["N39d29.34", "W87d14.94"],
+        "TVT"  : ["N40d27.48", "W82d07.61"],
+        "TYOTA": ["N38d42.56", "W85d22.22"],
+        "VACUB": ["N39d05.62", "W84d41.02"],
+        "VHP"  : ["N39d48.88", "W86d22.05"],
+        "VXV"  : ["N35d54.29", "W83d53.68"],
+        "WADAL": ["N39d11.50", "W83d58.12"],
+        "YRK"  : ["N38d38.65", "W82d58.70"],
+        "ZABAL": ["N39d32.12", "W85d55.09"],
+        "ZEGUL": ["N38d49.74", "W84d35.13"],
+        "ZEKUS": ["N40d42.13", "W85d06.48"],
+        "ZETIP": ["N38d40.57", "W84d46.88"],
+        "ZETLA": ["N38d55.70", "W84d44.62"],
+        "ZUGLU": ["N38d55.54", "W84d45.18"]
+    },
+    "restricted": [
```

Comment:
Would the JPG MOAs be of any use being added here? They are applicable daily and some do intersect the airspace